UCR/rdf-star-ucr.html

```diff
@@ -43,26 +43,14 @@ <h2>Introduction</h2>
         <a href="https://www.w3.org/community/rdf-dev/">W3C RDF Community Group</a>
         on the <a href="https://lists.w3.org/Archives/Public/public-rdf-star/"
                   >RDF* mailing list</a>
-        and <a href="https://github.com/w3c/rdf-star/issues/29">issue 29</a>
+        and <a href="https://github.com/w3c/rdf-star/issues?q=is%3Aissue+is%3Aopen+label%3Ause-case">issues</a>
         of the  community's GitHub repository.
     </section>
 
     <section>
       <h2>Contributing Use Cases and Requirements</h2>
       <p>
-        Add a comment to 
```

Comment:
My preference is leave there text about WHO-WHAT-WHY and also have the template link. i.e. Explanation and indication what is about to happen and the active link.

Reply:
I just put it back